widget/embedded/src/types/config.ts

```diff
@@ -219,4 +219,9 @@ export type WidgetConfig = {
   __UNSTABLE_OR_INTERNAL__?: {
     walletConnectListedDesktopWalletLink?: string;
   };
+
+  // Note: It is advised against adding this to the configuration.
+  __INTERNAL_DONT_USE__?: {
```

Comment:
NIT: I think it is better to use a more appropriate and clear name and comment.

suggestion:

```
// Internal configuration options. Not intended for public use or modification.
_internalSettings?: {
  autoUpdateSettings?: boolean; // If true, settings will be updated automatically based on the configuration.
};
```